srcpkgs/wren/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'wren'
+pkgname=wren
+version=0.3.0
+revision=1
+build_wrksrc=./projects/make
```

Comment:
Same thing here and for most other values you are using. No need to use `./` for paths :)